about.html

```diff
@@ -0,0 +1,191 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Visit Uganda Summit</title>
+    <link rel="stylesheet" href="about.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=lato"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <div class="mobile-menu" id="mobile-menu">
+      <div class="mobile-menu-header">
+        <button type="button" class="close-button">&times;</button>
+      </div>
+      <nav>
+        <a href="./index.html" class="navigate"> Home </a>
+        <a href="./about.html" class="navigate"> About </a>
+        <a href="./index.html#program" class="navigate"> Program </a>
+        <a href="#" class="navigate"> Join </a>
+        <a href="./about.html#sponsor" class="navigate"> Sponsor </a>
+      </nav>
+    </div>
+    <header>
+      <div class="social-media">
+        <nav class="social-nav">
+          <a href="https://facebook.com/edith.naisanga"
+            ><img src="./images/facebook-icon.png" alt="facebook"
+          /></a>
+          <a href="https://twitter.com/edithnaisanga"
+            ><img src="./images/twitter-icon.png" alt="twitter"
+          /></a>
+          <a href="#">English</a>
+          <a href="https://github.com/nedith">My page</a>
+          <a href="#">Logout</a>
+        </nav>
+      </div>
+      <div class="navbar">
+        <div class="navbar-logo-div">
+          <a href="./index.html"
+            ><img
+              class="summit-logo"
+              src="./images/logo.jpg"
+              alt="Visit Uganda logo"
+          /></a>
+        </div>
+
+        <nav class="navigation">
+          <a href="./about.html">About</a>
```

Comment:
- [ ] [Optional] It would be nice to change the link to `Home` instead of `About` when you are on the About Page. 👍